homeassistant/components/media_player/itunes.py

```diff
@@ -0,0 +1,260 @@
+"""
+homeassistant.components.media_player.itunes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Provides an interface to iTunes-API (https://github.com/maddox/itunes-api)
+
+Configuration:
+
+To use iTunes you will need to add something like the following to
+your configuration.yaml file.
+
+media_player:
+  platform: itunes
+  name: iTunes
+  host: http://192.168.1.16
+  port: 8181
+
+Variables:
+
+name
+*Optional
+The name of the device.
+
+url
+*Required
+URL of your running version of iTunes-API. Example: http://192.168.1.50:8181
+
+"""
+import logging
+
+from homeassistant.components.media_player import (
+    MediaPlayerDevice, MEDIA_TYPE_MUSIC, SUPPORT_PAUSE, SUPPORT_SEEK,
+    SUPPORT_VOLUME_SET, SUPPORT_VOLUME_MUTE, SUPPORT_PREVIOUS_TRACK,
+    SUPPORT_NEXT_TRACK)
+from homeassistant.const import (
+    STATE_IDLE, STATE_PLAYING, STATE_PAUSED)
+
+try:
+    import requests
```

Comment:
No need for the `try`? or no need for the import? It def doesn't work without the import there.